ci-operator/step-registry/redhat-chaos/application-outages/console/redhat-chaos-application-outages-console-commands.sh

```
@@ -49,5 +49,9 @@ rc=$?
 if [[ $TELEMETRY_EVENTS_BACKUP == "True" ]]; then
     cp /tmp/events.json ${ARTIFACT_DIR}/events.json
 fi
+
+if [[ -f /home/krkn/kraken/kraken.report.pdf ]]; then
+  cp /home/krkn/kraken/kraken.report.pdf ${ARTIFACT_DIR}/kraken.report.pdf
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)